packages/slate-react/src/utils/clone-fragment.js

```diff
@@ -32,7 +35,7 @@ function cloneFragment(event, value, fragment = value.fragment) {
 
   // Make sure attach is a non-empty node, since empty nodes will not get copied
   contents.childNodes.forEach(node => {
-    if (node.innerText.trim() !== '') {
+    if (node.innerText && node.innerText.trim() !== '') {
```

Comment:
This was [added recently](https://github.com/ianstormtaylor/slate/pull/1748/) and throws for plain text since they don't have `node.innerText`.

Reply:
Great catch!
Sorry for that =/ Seems like I'm not using any plain texts in my projects.